### PR TITLE
Use eslint-config-stripes

### PR DESCRIPTION
**Pull request by cherewaty**
Switches to using shared FOLIO eslint config: https://github.com/folio-org/eslint-config-stripes

### CODE REVIEW

File: src/components/contributors-list.js [hunk 1]
```diff
@@ -14,16 +14,16 @@ export default function ContributorsList({ data }) {
     <div>
       {Object.keys(contributorsByType).map((key) => {
         let names = contributorsByType[key];
-        key = capitalize(key);
+        let capitaliedKey = capitalize(key);
```
Comment (wwilsman) [inline src/components/contributors-list.js]: Missing `z`

Comment (cherewaty) [reply src/components/contributors-list.js]: Gross. Fixed.

**Comment by sadatay** [comment]
i'm down with this 👍

**Comment by cherewaty** [comment]
Need to merge https://github.com/thefrontside/ui-eholdings/pull/79 first.

**Pull request merged**